spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,36 @@
+describe AdminUser do
+  describe '.find_or_create' do
+    let(:admin_user) { AdminUser.find_or_create }
+    it 'returns an instance of AdminUser' do
+      expect(admin_user).to be_an_instance_of(AdminUser)
+    end
+    it 'ensures the returned AdminUser api_key is not null' do
+      expect(admin_user.api_key).not_to be(nil)
+    end
+    context 'when there is no existing AdminUser' do
+      it 'will create a new AdminUser' do
```

Comment:
In general, use present tense for the descriptions ("creates" instead of "will create").